includes/wc-core-functions.php

```diff
@@ -673,7 +673,7 @@ function get_woocommerce_currency_symbols() {
 			'ARS' => '&#36;',
 			'AUD' => '&#36;',
 			'AWG' => 'Afl.',
-			'AZN' => 'AZN',
+			'AZN' => '&#8916',
```

Comment:
Though it looks very similar, I'm not quite sure this is the correct symbol ... `&#8916;` seems to be the mathematical "pitchfork" operator `⋔`.

Looking at [CLDR R40](https://github.com/unicode-org/cldr/blob/release-40/common/main/az.xml#L5192) it looks like we should use [`&#8380;` ("Manat Sign" ₼)](https://unicode-table.com/en/20BC/). Also, we should add a semi-colon to the end:

```suggestion
			'AZN' => '&#8380;',
```

What do you think?

Reply:
Hi @SamirMahmudzade—just checking to make sure you saw my previous question and confirm if you are interested in moving this one forward?